mta-sts.md

```diff
@@ -302,7 +302,7 @@ sender fetches a HTTPS resource (policy) from a host at `policy.mta-sts` in the
 Policy Domain. The policy is served from a "well known" URI:
 `https://policy.mta-sts.example.com/.well-known/mta-sts/current`. To consider 
```

Comment:
Good point. I think there's no reason a reader should know this. ;)

On Thu, Sep 8, 2016 at 11:24 PM Security Sauce <notifications@github.com>
wrote:

> In mta-sts.md
> <https://github.com/mrisher/smtp-sts/pull/95#discussion_r78091906>:
>
> > @@ -302,7 +302,7 @@ sender fetches a HTTPS resource (policy) from a host at `policy.mta-sts` in the
> >  Policy Domain. The policy is served from a "well known" URI:
> >  `https://policy.mta-sts.example.com/.well-known/mta-sts/current` <https://policy.mta-sts.example.com/.well-known/mta-sts/current>. To consider
>
> aah, i would suggest to mention RFC5785 as reference (ideally closer to
> when we mention .well-known for the first time.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mrisher/smtp-sts/pull/95/files/4d95a5bc2dc4403de0cb1559707d523ce3db60d4#r78091906>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1vi4vE1NLVYX1g8Kl4qjAHiH2QKbN8ks5qoH0ggaJpZM4Jp_XD>
> .
>